src/frame.c

```diff
@@ -5455,6 +5433,18 @@ fget_output_method(const struct frame *f)
   return f->output_method;
 }
 
+unsigned
```

Comment:
You could return bool here. SImplify code elsewhere. Or handle in the method. Either way.